lmdeploy/pytorch/models/gemma.py

```diff
@@ -56,7 +56,8 @@ def __init__(self,
         if hasattr(config, 'query_pre_attn_scalar'):
             self.scaling = config.query_pre_attn_scalar**-0.5
         if self.model_type == 'gemma3_text':
-            is_sliding = bool((layer_idx + 1) % config.sliding_window_pattern)
+            sliding_window_pattern = getattr(config, 'sliding_window_pattern', 6)
```

Comment:
The output is wrong with the `gemma-3-4b-pt`
```
CUDA_VISIBLE_DEVICES=6,7 lmdeploy serve api_server /mnt/140/huggingface_hub/hub/modeloogle--gemma-3-4b-pt/snapshots/158684e1ce4f3b7cb12a57a4c762ae881d421b78 --backend pytorch --enable-prefix-caching --tp 1 --log-level INFO
```

Reply:
main branch is wrong too.